sysdig/resource_sysdig_secure_cloud_auth_account_test.go

```diff
@@ -317,3 +317,63 @@ func secureAzureCloudAuthAccountWithFCThreatDetection(accountID string) string {
 			}
 		}`, accountID, randomTenantId)
 }
+
+func TestGCPAgentlesScanningOnboarding(t *testing.T) {
+	rText := func() string { return acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum) }
+	accID := rText()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			if v := os.Getenv("SYSDIG_SECURE_API_TOKEN"); v == "" {
+				t.Fatal("SYSDIG_SECURE_API_TOKEN must be set for acceptance tests")
+			}
+		},
+		ProviderFactories: map[string]func() (*schema.Provider, error){
+			"sysdig": func() (*schema.Provider, error) {
+				return sysdig.Provider(), nil
+			},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: getResourceForGCPAgentlessScan(accID),
+			},
+			{
+				ResourceName:            "sysdig_secure_cloud_auth_account.gcp-agentless-scanning",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"component"},
```

Comment:
would be nice to understand thecontext on this part
https://github.com/sysdiglabs/terraform-provider-sysdig/pull/481/files

Reply:
resolved